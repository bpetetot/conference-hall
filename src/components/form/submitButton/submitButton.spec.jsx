/* eslint-env jest */
import React from 'react'
import SubmitButton from './index'

const props = {
  value: 'value',
  name: 'name',
}

describe('components/submitButton', () => {
  it('should render', () => {
    expect(
      <SubmitButton {...props}>
        <span>children</span>
      </SubmitButton>,
    ).toMatchSnapshot()
  })
})
