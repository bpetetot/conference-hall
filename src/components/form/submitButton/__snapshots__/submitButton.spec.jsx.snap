// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/submitButton should render 1`] = `
<SubmitButton
  invalid={false}
  loadingMessage="Saving..."
  name="name"
  pristine={false}
  secondary={false}
  submitting={false}
  type="submit"
  value="value"
>
  <span>
    children
  </span>
</SubmitButton>
`;
